tutorials/packaging/continuous-snap-delivery-from-circle-ci/continuous-snap-delivery-from-circle-ci.md

```diff
@@ -0,0 +1,497 @@
+Track    Arch    Channel    Version    Revision
+latest   amd64   stable     -          -
+                 candidate  -          -
+                 beta       -          -
+                 edge       2.10       1
+
+```
+
+You may have also received an email message from Circle CI notifying you about the successful execution, and another from the store notifying you about the new snap.
+
+Congratulations! Now you can spread the word and tell everybody to help you test your app by running a single command:
+
+
+```bash
+$ snap install hello-kyrofa --edge
+```
+
+## Now let’s do some crowd testing!
+Duration: 1:00
+
+From now on, every single change that lands in the `master` branch of your project will be made available to early adopters almost immediately. Your community of testers will be very happy because their edge snap will be updated automatically; they won't have to do anything complex to help you test your most recent changes. It's very likely that your community will grow, because they can be more involved with what's happening in your project; they can report any issues as soon as they appear, and contribute to improve your releases. After letting this crowd testing happen for a while, you can then push to lower risk channels until you are confident enough to make it available to the world in the `stable` channel.
+
+
+### Final code
+
+Your final code directory should contain a `.circleci/` directory containing both a `config.yml` as well as a `credentials.enc`, similar to [this demo repository].
+
+You should successfully have built your snap in a clean environment, configured your project to build the snap continuously on Circle CI, and deliver a new version to the edge channel for every change on your `master` branch. You can relax because your delivery pipeline is all automated. Now let your community know about this, encourage them to try the edge snap, and to tell their friends how cool it is to get even more testers.
+
+### Next steps
+
+  - Circle CI is very nice, but it can only build snaps for the amd64 architecture. If you need to deliver your snaps for users in other architectures like armhf and arm64, you can [use the snapcraft build service].
+  - Learn some more advanced techniques on how to use snaps by looking for other [tutorials]!
+  - Join the snapcraft.io community by subscribing to [our forum].
+
+### Further readings
+
+  - The [Circle CI 2.0 docs] have a lot of information about adding tests and other things to your CI executions.
+
+[Create your first snap]: /tutorial/create-your-first-snap
+[https://github.com]: https://github.com/
+[https://github.com/new]: https://github.com/new
+[https://dashboard.snapcraft.io]: https://dashboard.snapcraft.io/
+[four channels]: https://snapcraft.io/docs/reference/channels#channels
+[this demo repository]: https://github.com/kyrofa/hello-snap-kyrofa
+[use the snapcraft build service]: https://build.snapcraft.io/
+[tutorials]: https://tutorials.ubuntu.com
```

Comment:
This one could be changed to: https://tutorials.ubuntu.com?q=snap

Reply:
Very good, fixed.